### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aliasgharbagheri**
ignore all file public, assets, logs and file assets _next to nextjs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,5 @@ build/
 .env
 .env*.local
 
-log/
```
Comment (hosseiniahb) [inline .gitignore:10]: not remove .log files, becuase .log file mass storage
you add log file and commit

**Review by hosseiniahb — COMMENTED** [review]
add .log

File: .gitignore (hunk 1 above)
Comment (aliasgharbagheri) [reply .gitignore:10]: ok

Comment (hosseiniahb) [reply .gitignore:10]: ok merge with main branch

Comment (aliasgharbagheri) [reply .gitignore:10]: ok!

Comment (aliasgharbagheri) [reply .gitignore:10]: my adding log files

Comment (hosseiniahb) [reply .gitignore:10]: ok my merge to main branch

**Review by hosseiniahb — APPROVED** [review]

**Pull request merged**